language/polkavm/move-to-polka/src/stackless/module_context.rs

```diff
@@ -694,8 +694,19 @@ impl<'mm: 'up, 'up> ModuleContext<'mm, 'up> {
                 }
             }
             Type::Vector(_) => Some(self.rtty_cx.get_llvm_type_for_move_vector(self, tyvec)),
-            Type::Tuple(_) => {
-                todo!("{mty:?}")
+            Type::Tuple(types_vec) => {
+                let llvm_types = types_vec
+                    .iter()
+                    .map(|move_ty| {
```

Comment:
Question: Why is it variable called `move_ty`?

Reply:
Well because its move type, which we transform into LLVM type. Regarding naming itself - I will go with full name move_type, but was just following rest of the code style.

Reply:
Okay, I guessed it was ty for for type. I just wanted to double check :+1: